client/my-sites/customer-home/components/celebrate-launch-modal.jsx

```diff
@@ -1,13 +1,27 @@
-import { ConfettiAnimation, Gridicon } from '@automattic/components';
+import { Gridicon, ConfettiAnimation } from '@automattic/components';
 import { Button, Modal } from '@wordpress/components';
 import { useTranslate } from 'i18n-calypso';
-import { useEffect } from 'react';
+import { useEffect, useState, useRef } from 'react';
+import ClipboardButton from 'calypso/components/forms/clipboard-button';
+import Tooltip from 'calypso/components/tooltip';
+import { useGetDomainsQuery } from 'calypso/data/domains/use-get-domains-query';
 import { omitUrlParams } from 'calypso/lib/url';
+import { createSiteDomainObject } from 'calypso/state/sites/domains/assembler';
 
 import './celebrate-launch-modal.scss';
 
 function CelebrateLaunchModal( { setModalIsOpen, site } ) {
 	const translate = useTranslate();
+	const isPaidPlan = ! site?.plan?.is_free;
+	const isBilledYearly = site?.plan?.billing_period === 'Yearly';
```

Comment:
I was wondering if there was a better selector for this, I couldn't find anything obvious